packages/cli/cmd/box_exec.go

```diff
@@ -121,6 +124,7 @@ options:
 	// Add flags
 	cmd.Flags().BoolVarP(&opts.Interactive, "interactive", "i", false, "Enable interactive mode (with stdin)")
 	cmd.Flags().BoolVarP(&opts.Tty, "tty", "t", false, "Force TTY allocation")
+	cmd.Flags().StringVarP(&opts.WorkingDir, "work-dir", "w", "", "Working directory inside the box (default: use box config or /root)")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&opts.WorkingDir, "workdir", "w", "", "Working directory inside the box (default: use box config or /root)")
```

和 docker 一致吧 -w, --workdir string                   Working directory inside the container

Reply:
done，另外 interactive 那个参数改成 tty，我后面再提一组 PR 改，几个项目切来切去有点麻了